Mathlib/Logic/Basic.lean

```diff
@@ -923,6 +923,26 @@ theorem forall_prop_congr' {p p' : Prop} {q q' : p → Prop} (hq : ∀ h, q h 
   propext (forall_prop_congr hq hp)
 #align forall_prop_congr' forall_prop_congr'
 
+#align forall_congr_eq forall_congr
+
+lemma imp_congr_eq {a b c d : Prop} (h₁ : a = c) (h₂ : b = d) : (a → b) = (c → d) :=
+  propext (imp_congr h₁.to_iff h₂.to_iff)
+
+lemma imp_congr_ctx_eq {a b c d : Prop} (h₁ : a = c) (h₂ : c → b = d) : (a → b) = (c → d) :=
+  propext (imp_congr_ctx h₁.to_iff fun hc ↦ (h₂ hc).to_iff)
+
+lemma eq_true_intro (h : a) : a = True := propext (iff_true_intro h)
+lemma eq_false_intro (h : ¬a) : a = False := propext (iff_false_intro h)
+
+/-- Alias of `propext`. -/
+lemma Iff.eq : (a ↔ b) → a = b := propext
```

Comment:
```suggestion
alias Iff.eq := propext
```

Reply:
I can't because `alias` is not imported there. Should I import it?

Reply:
yes

Reply:
Nevermind, it is? 🤔

Reply:
Sorry I remember the problem: `alias` creates `def Iff.eq := propext`, not `lemma Iff.eq := propext`. Will nolint for now.

Reply:
it shouldn't? Report it as a bug if so

Reply:
It's supposed to create defs from defs and theorems from theorems, but you are aliasing an `axiom` and I guess these can be either defs or theorems